editor/gui/scene_tree_editor.h

```diff
@@ -62,6 +62,7 @@ class SceneTreeEditor : public Control {
 	String filter_term_warning;
 
 	AcceptDialog *error = nullptr;
+	AcceptDialog *error2 = nullptr;
```

Comment:
Both can appear at the same time if the name is invalid it is replacing the invalids characters with `_`. This means if you try to rename a node that is Unique to `Node%` and you already have a unique name called `Node_` it will first tells you there was invalid characters and then tells you this unique name is already used. Right now this specific case make the unique name be revoked even with normal renames because I think it checked them in the wrong order.

I'm not sure about adding a return to the first error as this is not how it behaved previously.

Reply:
read my edit